docs/sources/features/explore/index.md

```diff
@@ -115,3 +115,16 @@ after switching to the Logs data source, the query changes to:
 `{job="grafana"}`
 
 This will return a chunk of logs in the selected time range that can be grepped/text searched.
+
+
+#### Live tailing
+
+With Loki you can use the Live tailing feature to see realtime logs as they are pushed to Loki.
```

Comment:
real-time should be hyphenated

Reply:
Instead of:

> With Loki you can use the Live tailing feature ...

Prefer:

> Use the Live tailing feature ...

Reply:
Changed `Loki` to `supported data sources` as not all support Live tailing.